letsencrypt/cli.py

```diff
@@ -127,9 +127,9 @@ def _determine_account(args, config):
         elif len(accounts) == 1:
             acc = accounts[0]
         else:  # no account registered yet
-            if args.email is None:
+            if args.email is None and not args.allow_unsafe_registration:
```

Comment:
so basically `--alow-unsafe-registration` will never prompt for email? that doesn't sound good